utilities/test/test_clog.cc

```diff
@@ -0,0 +1,229 @@
+#include <gtest/gtest.h>
+#include <memory>
+
+#include <utilities/clog.h>
+#include <utilities/test_fixture.h>
+
+namespace {
+
+static const char *cLoggerName{"TestClog"};
+
+class TestClog : public qle::TestFixture {
+ protected:
+  void SetUp() { qle::Logger::set_log_level(qle::Logger::LogLevel::TRACE); }
+  void TearDown() { qle::Logger::set_log_level(qle::Logger::LogLevel::TRACE); }
```

Comment:
If updating `set_log_level` to check invalid arguments. this should be updated. A unit test might required